spec/ruby/library/bigdecimal/round_spec.rb

```diff
@@ -187,4 +187,10 @@
       @n2_49.round(0, BigDecimal::ROUND_HALF_EVEN).should == @neg_two
     end
   end
+
+  it 'raise exception, if self is special value' do
+    lambda { BigDecimal('NaN').round }.should raise_error(FloatDomainError)
+    lambda { BigDecimal('Infinity').round }.should raise_error(FloatDomainError)
+    lambda { BigDecimal('-Infinity').round }.should raise_error(FloatDomainError)
+  end
```

Comment:
The split is done automatically, but it's niver to split because then the commit message tend to be right and I don't have to remove [Truffle] and impl details unrelated to the spec.